Forks project run  `Push latest Docker image` action every day

### Describe the current behavior

Every day at 5:41am GMT a `Push latest Docker image` action triggers in my fork.
It fails and notifies me.

Some computation time that I don't want/need is wasted.

### Steps to reproduce

1. Fork the project in a github user other than gristlabs
2. Wait 24 hours
3. Check triggered actions

### Describe the expected behavior

The workflow can't be triggered automatically in forks.

*Proposed solution*
May be we can add a boolean variable in the CI so if not present or set at False the workflow doesn't trigger.

### Where have you encountered this bug?

- [ ] On [docs.getgrist.com](https://docs.getgrist.com)
- [ ] On a self-hosted instance

### Instance information (when self-hosting only)

Appears only in github CI/actions

In my fork It fails on the `Prepare image but not push it yet` step.
But as said, I haven't set any needed env variable.

Here are the logs:

```raw
#35 ERROR: process "/bin/sh -c yarn run build:prod" did not complete successfully: exit code: 2
------
 > [builder 14/17] RUN yarn run build:prod:
0.190 yarn run v1.22.19
0.226 $ buildtools/build.sh
0.240 Using extra app directory
0.240 + tsc --build tsconfig-ext.json
24.29 app/server/mergedServerMain.ts(74,3): error TS2741: Property 'edition' is missing in type 'Promise<IGristCoreConfig>' but required in type 'IGristCoreConfig'.
24.39 error Command failed with exit code 2.
24.39 info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
------
Dockerfile:36
--------------------
  34 |     COPY stubs /grist/stubs
  35 |     COPY buildtools /grist/buildtools
  36 | >>> RUN yarn run build:prod
  37 |     
  38 |     # Prepare material for optional pyodide sandbox
--------------------
ERROR: failed to solve: process "/bin/sh -c yarn run build:prod" did not complete successfully: exit code: 2
Error: buildx failed with: ERROR: failed to solve: process "/bin/sh -c yarn run build:prod" did not complete successfully: exit code: 2
```

--- a/.github/workflows/docker_latest.yml
+++ b/.github/workflows/docker_latest.yml
@@ -46,6 +46,7 @@ jobs:
   push_to_registry:
     name: Push latest Docker image to Docker Hub
     runs-on: ubuntu-latest
+    if: ${{ vars.RUN_DAILY_BUILD }}
     strategy:
       matrix:
         python-version: [3.11]

